plugin/EstateList.php

```diff
@@ -398,6 +404,17 @@ public function estateIterator($modifier = EstateViewFieldModifierTypes::MODIFIE
 			$this->_currentEstate['id'];
 		$this->_currentEstate['title'] = $currentRecord['elements']['objekttitel'] ?? '';
 
+		if ( $this->_pDataView instanceof DataDetailView ) {
+			ob_start();
```

Comment:
I think if there's a bug, it will come from here. I can use `echo("<script>location.href = '".$url."'</script>");` as a safer solution instead of `wp_redirect`, what do u think?

Reply:
@jmaas-onoffice

Reply:
The bug is that on one instance after upgrading, the detail page is broken. I need to save the detail page to make it work again. It might be related to the Enfold theme, so I'm trying to set up a test instance with it.

I will then see if the bug is related to what you are talking about here. Thanks for the hint!